Automattic/syntaxhighlighter
# Refactor block JS code

Refactor the plugin's block editor code to make it easier to work with.

## Changes proposed: 

* Break up block definition into multiple files
* Update syntax & formatting
* Update block setting items to JSX syntax

## Testing instructions:

* Open or create a post with a SyntaxHighlighter Code block
* Make sure it works in the editor & frontend, including various settings
